tests/integration_test/grpc_deadline_test/grpc_deadline_test.go

```diff
@@ -77,22 +77,23 @@ plans {
   }
 }`,
 		},
-		{
-			desc: "Success after 20s because ESPv2 automatically disables response timeouts for streaming RPCs",
-			testPlan: `
-plans {
-  echo_stream {
-    call_config {
-      api_key: "this-is-an-api-key"
-    }
-    request {
-      text: "Hello, world!"
-      response_delay: 20
-    }
-    count: 1
-  }
-}`,
-		},
+		// TODO(b/185919750): Adjust the integration tests due to the introduced newly-involved max_stream_timeout.
```

Comment:
Well if these tests caught the error, won't this affect our customers as well? It changes the behavior the customers expect.

What is the failure message for these tests?

Reply:
Our Istio tests noted the change from 504 to 408 and our tests are now failing. See comment https://github.com/istio/istio/issues/31616#issuecomment-824156192 as to if this was expected and documented.

Reply:
I think after setting a static large `max_stream_timeout` can workaround. The issue now is the error code is flaky between 408 and 504.

Reply:
I guess the actual error code does not matter, as we encourage users to not depend on the exact error.

https://cloud.google.com/apis/design/errors

> Error messages are not part of the API surface. They are subject to changes without notice. Application code must not have a hard dependency on error messages.

@TAOXUY can you change the failing integration tests to just check an error occurred, without checking the specific error? Thanks for researching and setting the static `max_stream_timeout`.

Reply:
ACK on not depending on the error.